webpack.config.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path');
+
+module.exports = {
+
+    mode: 'development',
+
+    entry: './way_to_home/static/index.js',
```

Comment:
use path.join
The entry point path should be `way_to_home/static/src/app.js`

Reply:
fixed